test/camunda-cloud/integration/bpmnlint/start-event-required.spec.js

```diff
@@ -0,0 +1,80 @@
+const { expect } = require('chai');
+
+const Linter = require('bpmnlint/lib/linter');
+
+const NodeResolver = require('bpmnlint/lib/resolver/node-resolver');
+
+const { readModdle } = require('../../../helper');
+
+const versions = [
+
+  //   '1.0',
+  //   '1.1',
+  //   '1.2',
+  //   '1.3',
+  //   '8.0',
+  //   '8.1',
+  //   '8.2',
+  //   '8.3',
+  //   '8.4',
+  //   '8.5',
+  //   '8.6',
+  '8.7',
```

Comment:
Preserving currently supported Camunda versions:

```suggestion
  '8.5',
  '8.6',
  '8.7'
```